client/src/components/CommonComponents/Modal.js

```diff
@@ -0,0 +1,47 @@
+import React, { Fragment } from "react";
+import { Typography, Modal, Box } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import "./../../App.css";
+const useStyles = makeStyles(theme => ({
+  paper: {
+    position: "absolute",
+    backgroundColor: theme.palette.background.paper,
+    boxShadow: theme.shadows[5],
+    padding: theme.spacing(4),
+    outline: "none"
+  }
+}));
+export default function CustomModal(props) {
+  const classes = useStyles();
+
+  return (
+    <Fragment>
+      <Modal
+        aria-labelledby="simple-modal-title"
+        aria-describedby="simple-modal-description"
+        open={props.open}
+        onClose={() => {
```

Comment:
```suggestion
        onClose={props.handleClose}
```

Reply:
Is there is particular reason to use only arrow functions and not normal function